production/PRODUCTION_README.md

```diff
@@ -1,10 +1,289 @@
+
+The instantiate and deploy commands can be tested before instantiation with the 'dry-run' option:  
+
+- go-deploy --workspace noctua-production-YYYY-MM-DD --working-directory aws -verbose -dry-run  --conf config-stack.yaml
+   
+The instance name of the server will be noctua-production-YYYY-MM-DD.geneontology.org
+
+A barista instance will be created on go-barista-production-YYYY-MM-DD.geneontology.org
+
+
+For testing
+- go-deploy --workspace ___-noctua-test-YYYY-MM-DD --working-directory aws -verbose --conf config-instance.yaml
+
+- go-deploy --workspace ___-noctua-test-YYYY-MM-DD --working-directory aws -verbose --conf config-stack.yaml
+
+- go-deploy --workspace ___-noctua-test-YYYY-MM-DD --working-directory aws -verbose -destroy
+
+The deploy command can be tested before instantiation with the 'dry-run' option:  
+
+- go-deploy --workspace ___-noctua-test--YYYY-MM-DD --working-directory aws -verbose -dry-run  --conf config-stack.yaml
+
+   
+The instance name of the server will be ___-noctua-test-YYYY-MM-DD.geneontology.io
+
+A barista instance will be created on ___-barista-test-YYYY-MM-DD.geneontology.io
+
+
+Copy the above commands into a text editor and update the workspace names.     
+
+  
+
+
+## Workflow
+In order to ensure reproducibility, a Docker development environment is created locally with the required tools for deployment. Configuration files are updated. These configuration files are used by Terraform and Ansible to instantiate and configure the instances on aws. Once an instance is created on aws, artifacts are deployed to a staging directory on the instance.  Followed by deployment. 
+
+1.  Create a Docker development environment and clone repository from Github
+2.  Update credentials for accessing and provisioning resources on aws
+3.  Add an entry in the aws s3 for storing information about the workspace being created and initialize
+4.  Update configuration file to instantiate instance on aws
+5.  Instantiate instance on aws
+6.  Update configuration files to deploy on aws
+7.  Deploy instance
+
+
+
+### 1.  Create a Docker development environment and clone repository from Github
+We have a docker based dev environment with all these tools installed. See last section of this README (Appendix I: Development Environment).
+See Prerequisites 4 for docker 
+The instructions in this document are run from the POV that they are executed within the developement environment; i.e.:
+```
+docker run --name go-dev -it geneontology/go-devops-base:tools-jammy-0.4.2  /bin/bash
+git clone https://github.com/geneontology/noctua_app_stack.git
+cd noctua_app_stack
+go-deploy -h
+```
+
+### 2.  Update credentials for accessing and provisioning resources on aws
+####  Copy the ssh keys from your docker host into the running docker image, in `/tmp`:
+
+These commands may have to be executed from a separate terminal that can run Docker commands.
+See Prerequisites 2 for keys and copy keys from SpderOak
```

Comment:
Could you remove the reference to SpiderOak here? We'll do that for developer onboarding (hopefully). I'd ideally like to avoid some specifics here. Paranoia.

Reply:
Agreed - Security is paramount.  I have updated the documentation to remove reference to SpiderOak